microsoft-edge/webview2/get-started/winui.md

```diff
@@ -415,8 +435,12 @@ Congratulations, you built your first WebView2 app!
 <!-- ====================================================================== -->
 ## WinUI 3 WebView2 special considerations
 
+
+<!-- ------------------------------ -->
 #### SmartScreen
 
+_todo: this section is now out of date.  We no longer recommend using environment variables here; instead, use a custom `CoreWebView2Environment`._
```

Comment:
how to SmartScreen? @david-risney 

Rendered section in PR:
https://review.learn.microsoft.com/microsoft-edge/webview2/get-started/winui?branch=pr-en-us-3104#smartscreen

Reply:
The code would be something like this:

```cs
            CoreWebView2EnvironmentOptions environmentOptions = new CoreWebView2EnvironmentOptions();
            environmentOptions.AdditionalBrowserArguments = "--disable-features=msSmartScreenProtection";

            string browserFolder = null; // Use null to get default browser folder
            string userDataFolder = null; // Use null to get default user data folder
            CoreWebView2Environment environment = await CoreWebView2Environment.CreateWithOptionsAsync(
                browserFolder, userDataFolder, environmentOptions);

            // ...

            this.WebView2.EnsureCoreWebView2Async(environment);
```

Reply:
added code, updated section